### ISSUE TITLE
Mini-player: Position of track & artist names..

### ISSUE THREAD

**Issue by Rich28** [digimezzo/dopamine-windows#556]
So glad I stumbled upon Dopamine - what a great player.  This is a minor point.  At times, when using the mini-player the track and artist names are positioned over text which features in the album artwork.  Is there any chance that the track and artist names could be repositioned, so that this info is displayed let's say beneath the album artwork?

**Comment by digimezzo** [comment]
Could you attach a screenshot that illustrates this issue?

**Comment by Rich28** [comment]
Yep, can do.  Apologises I should have done this in my initial post. (Incidentally, I've just upgraded to version 1.53 and gather that the issue I'm referring to actually relates to playback via the cover player).
 
![dopamine](https://user-images.githubusercontent.com/33752719/33180171-be884ca4-d063-11e7-95be-49f8b32c9440.png)
